ui/packages/catalog-ui-search/src/main/webapp/component/query-src/query-src.view.js

```diff
@@ -16,12 +16,46 @@
 const Marionette = require('marionette')
 const _ = require('underscore')
 const $ = require('jquery')
-const template = require('./query-src.hbs')
 const CustomElements = require('js/CustomElements')
 const sources = require('component/singletons/sources-instance')
+import React from 'react'
 
 module.exports = Marionette.ItemView.extend({
-  template: template,
+  template(props) {
+    return (
+      <React.Fragment key={Math.random()}>
+        <div className="choice is-all is-available">
+          <span className="choice-text">All Sources</span>
+          <span className="choice-selected fa fa-check" />
+        </div>
+        {props.map(source => {
+          return (
```

Comment:
✏️ 	It might be cleaner / more readable to factor this out into it own helper function